src/newlines.cpp

```diff
@@ -2205,6 +2207,30 @@ static void newlines_brace_pair(chunk_t *br_open)
 
          if (is_it_possible)
          {
+            // Issue 2795
```

Comment:
This only is usefull when we actually have the option set, so move it into a `if (  options::code_width() > 0)` block

Reply:
NO!
Only after making some lines to a one liner, the user wanted:
_nl_create_func_def_one_liner = true_
we are able to know if the line is bigger as _code_with_
Here we have:
_code_width  = 140_
We need a copy to be able to restore the code.
We mark the opening brace with _PCF_NOT_POSSIBLE_ to do the job
only once.

Reply:
Whats the purpose of the vector content if `code_with` is not set? `code_with = 0`